advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md

```diff
@@ -58,4 +58,4 @@ This section contains questions for you to check your understanding of this less
 
 ### Additional resources
 
-This section contains helpful links to other content. It isn’t required, so consider it supplemental.
+Expand this section to have more resources on github
```

Comment:
When a lesson has an additional resources section that doesn't contain any resources, it must follow our styleguide and contain the following content:

```markdown
This section contains helpful links to related content. It isn’t required, so consider it supplemental.

- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
```

Reply:
ok sir
i will update as you said

Reply:
i will try to correct the linting errors